Source/Urho3D/CSharp/Scripts/Compiler.cs

```diff
@@ -75,18 +75,22 @@ public static Assembly Compile(IEnumerable<string> sourceFiles, IEnumerable<stri
                     string resourceName = error.FileName;
                     foreach (string resourceDir in Context.Instance.Cache.ResourceDirs)
                     {
-                        if (resourceName.StartsWith(resourceDir))
+                        // ressource paths use lower key c: and '/' as separator and but Windows uses upper key C: and '\' 
+                        if ( resourceName.ToLowerInvariant().Replace('\\', '/').StartsWith( resourceDir.ToLowerInvariant().Replace('\\', '/') ))
```

Comment:
This may break when case-sensitive filesystem is used. Only disk letter should be normalized for comparison.